utils.groovy

```diff
@@ -37,12 +37,12 @@ def rsync(from, to) {
         return
     }
 
-    def rsync_key = readFile(file: rsync_keypath)
-    rsync_key = rsync_key[0..12]
-
     shwrap("""
     # so we don't echo password to the jenkins logs
-    set +x; export RSYNC_PASSWORD=${rsync_key}; set -x
+    set +x
+    RSYNC_PASSWORD=\$(cat ${rsync_keypath})
+    export RSYNC_PASSWORD=\${RSYNC_PASSWORD:0:13}
```

Comment:
Does  rsync_key[0..12] and ${RSYNC_PASSWORD:0:13} gives same length of string?

Reply:
Yeah it's kinda weird. In Groovy, the end index is inclusive, but in bash, it's exclusive.

Reply:
+1